docs/resources/notification_policy.md

```diff
@@ -108,10 +108,19 @@ Optional:
 - `limit` (Set of String) A numerical limit. Example: `100`.
 - `pool_id` (Set of String) Load balancer pool identifier.
 - `product` (Set of String) Product name. Available values: `worker_requests`, `worker_durable_objects_requests`, `worker_durable_objects_duration`, `worker_durable_objects_data_transfer`, `worker_durable_objects_stored_data`, `worker_durable_objects_storage_deletes`, `worker_durable_objects_storage_writes`, `worker_durable_objects_storage_reads`.
-- `services` (Set of String)
+- `services` (Set of String) A list of clickhouse services to alert on.
 - `slo` (Set of String) A numerical limit. Example: `99.9`.
 - `status` (Set of String) Status to alert on.
 - `zones` (Set of String) A list of zone identifiers.
+- `requests_per_second` (Set of String) Requests per second threshold for dos alert.
```

Comment:
@jacobbednarz I am assuming this will auto update https://github.com/cloudflare/terraform-provider-cloudflare/pull/1830/files#diff-4bafef06f0138d4dc0de92a6747d56230f6412a454bc6c36cb3c3c599816bb16R177

Please correct me if I am wrong. I can update manually.

Reply:
other way around :) https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md

you write the schema and functionality to  support it, run `make docs` and then it generates the documentation.

Reply:
oh my bad. Will update it